tests/integration/codebuild/buildspec.os.amazonlinux.2023.yml

```diff
@@ -0,0 +1,113 @@
+version: 0.2
+
+env:
+  variables:
+    OS_DISTRIBUTION: amazonlinux
+    PYTHON_LOCATION: "/usr/local/bin/python3"
+    TEST_NAME: "aws-lambda-python-rtc-amazonlinux-test"
+batch:
+  build-matrix:
+    static:
+      ignore-failure: false
+      env:
+        privileged-mode: true
+    dynamic:
+      env:
+        variables:
+          DISTRO_VERSION:
+            - "2023"
```

Comment:
We should be testing with `2023-minimal`

Reference : https://github.com/aws/aws-lambda-base-images/issues/92#issuecomment-1737531040